src/style.css

```
@@ -1 +1,37 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin: 0 4px 0 0;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n + 2) {
  background-image: url(images/star-active.svg);
}
```
You can unite selectors